.gitignore

```diff
@@ -3,3 +3,6 @@ typesafe_conductr_cli.egg-info
 __pycache__
 deb_dist/*
 !deb_dist/Dockerfile
+.idea
+*.iml
```

Comment:
It tend to add IDE specific ignores to the global ignore file. Saves lots of typing. :)